fcrepo-service-activemq/pom.xml

```diff
@@ -76,6 +76,11 @@
     </dependency>
 
     <dependency>
+      <groupId>xerces</groupId>
```

Comment:
Well, we're using jena-osgi, which doesn't contain Xerces, since Karaf and such systems already include an implementation for that. That means that in the non-pax-exam-based test, we get a ClassNotFound exception unless xerces is included in test scope. The error doesn't affect the running code or the tests, but it does generate an ugly stack trace.